android/java/org/chromium/chrome/browser/settings/BraveSyncScreensPreference.java

```diff
@@ -1284,46 +1282,31 @@ public void run() {
                   } else {
                       String qrCodeString = getBraveSyncWorker().GetQrDataJson(seedHex);
                       assert qrCodeString != null && !qrCodeString.isEmpty();
-                      fillQrCode(qrCodeString);
+                      ChromeBrowserInitializer.getInstance().runNowOrAfterFullBrowserStarted(
```

Comment:
just to learn more : does this function also check for native initialisation ?

Reply:
yes, `runNowOrAfterFullBrowserStarted` is executed after  [handlePostNativeStartup](https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/java/src/org/chromium/chrome/browser/init/ChromeBrowserInitializer.java;drc=bad98f27515a6b830f87013c236bc6463c19ff08;bpv=1;bpt=1;l=240?gsn=handlePostNativeStartup&gs=kythe%3A%2F%2Fchromium.googlesource.com%2Fchromium%2Fsrc%3Flang%3Djava%3Fpath%3Dorg.chromium.chrome.browser.init.ChromeBrowserInitializer%23b278f0ae7cbe218e10bff389c2385e247ba0a955780134e47d60eb741218b173)  is executed, so native libs are initialized at that moment.